components/image/imagemagick/pkg5

```diff
@@ -20,6 +20,7 @@
         "library/openexr",
         "library/zlib",
         "print/filter/ghostscript/fonts/gnu-gs-fonts-std",
+        "shell/ksh93",
         "system/library",
         "system/library/freetype-2",
         "system/library/g++-7-runtime",
```

Comment:
pkg5 seem to need another update. You have remove library/fftw-3 from the REQUIRED_PACKAGES but it's still here.

Reply:
Sorry for that.  Fixed.